src/Evolution/Systems/GrMhd/ValenciaDivClean/QuadrupoleFormula.hpp

```diff
@@ -54,16 +55,19 @@ struct QuadrupoleMomentCompute
 /// $\gamma$ being the determinant of the spatial metric, \f$x\f$ the
 /// coordinates in Frame Fr, and \f$v\f$ the corresponding spatial velocity.
 template <typename DataType, size_t Dim, typename OutputCoordsTag,
-          typename Fr = Frame::Inertial>
+          typename VelocityTag, typename Fr = Frame::Inertial>
 struct QuadrupoleMomentDerivativeCompute
     : hydro::Tags::QuadrupoleMomentDerivative<DataType, Dim, Fr>,
       db::ComputeTag {
-  using argument_tags = tmpl::list<TildeD, OutputCoordsTag,
-                              hydro::Tags::SpatialVelocity<DataType, Dim, Fr>>;
+  using argument_tags = tmpl::list<TildeD, OutputCoordsTag, VelocityTag>;
 
   using base = hydro::Tags::QuadrupoleMomentDerivative<DataType, Dim, Fr>;
   using return_type = typename base::type;
 
+  static std::string name() {
+    return "QuadrupoleMomentDerivative" + db::tag_name<VelocityTag>();
```

Comment:
I find `QuadrupoleMomentDerivative(" + db::tag_name<VelocityTag>() + ")";` easier to read in input files. What do you think?